addons/website_forum/static/src/js/website_forum.js

```diff
@@ -164,6 +166,17 @@ publicWidget.registry.websiteForum = publicWidget.Widget.extend({
             }
         });
 
+        _.each(self.$('.o_wforum_bio_popover'), function (bio) {
+            var content = $(bio).find('.o_wforum_bio_popover_data').html();
+            $(bio).popover({
+                trigger: 'hover',
+                offset: 10,
+                animation: false,
+                html: true,
+                content: content,
+            });
+        });
```

Comment:
It's not editable but you're right, that div is never visible.
I'd suggest to create a template in `static/src/xml` and populate it with data-attributes as you suggested (the template is necessary because of the layout of the popup + classes etc..).  What do you think?

Reply:
@stefanorigano I am not sure to understand what you suggest. The content of that div is filled with server information, how would you do that with static xml? What I suggested was:

Insteaf of:
```xml
<div class="o_wforum_bio_popover">
    ...
    <div class="o_wforum_bio_popover_data d-none">
        Your awesome dynamic popover content
    </div>
</div>
```

You do:
```xml
<t t-set="popover_content">
    Your awesome dynamic popover content
</t>
<div class="o_wforum_bio_popover" t-att-data-content="popover_content">
    ...
</div>
```

That way you won't need JS to give the content to the popover library, this is automatic.

Reply:
mmm.. yes the xml will be super neat but what about the final DOM?
we'll add a lot of raw html as attribute value... are you sure?

Reply:
The final DOM will be cleaner: no extra HTML to parse on page rendering :wink:

Reply:
Ok, done 🙈 ^^
![Screenshot 2019-09-04 at 13 57 06](https://user-images.githubusercontent.com/8558351/64252795-20553600-cf1c-11e9-862c-dee3e057ea87.png)